Modules/Quotas/test/Quotas.Application.Tests/Tests/Identities/GetQuotasForIdentity/HandlerTests.cs

```diff
@@ -0,0 +1,63 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using AutoMapper;
+using Backbone.BuildingBlocks.Application.Abstractions.Infrastructure.UserContext;
+using Backbone.Modules.Quotas.Application.Identities.Queries.GetQuotasForIdentity;
+using Backbone.Modules.Quotas.Application.Infrastructure.Persistence.Repository;
+using Backbone.Modules.Quotas.Application.Metrics;
+using Backbone.Modules.Quotas.Domain;
+using Backbone.Modules.Quotas.Domain.Aggregates.Identities;
+using Backbone.Modules.Quotas.Domain.Aggregates.Metrics;
+using Backbone.Modules.Quotas.Domain.Aggregates.Tiers;
+using Backbone.Modules.Quotas.Domain.Metrics;
+using FakeItEasy;
+using FluentAssertions;
+using Xunit;
+using static Backbone.UnitTestTools.Data.TestDataGenerator;
+
+namespace Backbone.Modules.Quotas.Application.Tests.Tests.Identities.GetQuotasForIdentity;
+public class HandlerTests
+{
+    [Fact]
+    public async Task Returns_individual_and_tier_quotas_for_identity()
```

Comment:
This test doesn't test the probably most complex and most important logic: the calculation of the current usage.